add last-modified header to .js and .css

spanditcaa opened the pull request:
@wraithgar this PR sets a last-modified header on the .js and .css bundles (when not in development mode).  This lets us take advantage of browser caching and speeds up page loads, especially on mobile.

wraithgar commented:
Browser caching sure is hard. We'd expected the `expiresIn` value to handle this, but evidently you're not seeing that?

I'm all for this, a quick test should probably be added to this PR that ensures the last-modified doesn't change between requests when not on dev mode.

This would be a major semver release as this changes default behavior.

spanditcaa commented:
Updated.  I also found that many of the expect()'s were incompat with code 1.5's expect -- you'll notice I updated them, removing the descriptor which seems to be no longer supported.

wraithgar commented:
I see you also added a test to assert that the existing caching headers are present in production mode. Nice.

wraithgar commented:
published as `v6.0.0`

Thank you.

spanditcaa commented:
Thanks @wraithgar
